src/api/app/views/webui/comment/_new.html.erb

```diff
@@ -1,6 +1,8 @@
 <% unless User.current.is_nobody? %>
-  <%= save_comment_form do %>
-    <%= text_area_tag 'body', nil, :size => '80x4', :onkeyup => 'sz(this);', :onclick => 'sz(this);', :style => 'width: 90%;', :placeholder => 'Add a new comment (markdown markup supported)', :required => 'required' %>
-    <%= submit_tag 'Add comment'%>
-  <%end%>
+  <%= form_for :comment, url: { controller: :comments, action: :create }, method: :post do |f| %>
```

Comment:
Hm, as we anyway rewrite almost the whole view, why not use HAML?
http://htmltohaml.com/